string problemDirectory
string problemName
---
string data
